packages/core/src/core/index.ts

```
@@ -565,29 +660,92 @@ const ensureConnected = async (method: BaseMethod, pollingId: number) => {
-        device = initDevice({ connectId } as BaseMethod);
+export const cancel = (context: CoreContext, connectId?: string) => {
+  const { requestQueue, setPrePendingCallPromise } = context;
+  if (connectId) {
+    try {
+      // let device;
+      // if (DataManager.isBleConnect(env)) {
+      //   device = initDeviceForBle({ connectId } as BaseMethod);
+      // } else {
+      //   device = initDevice({ connectId } as BaseMethod);
+      // }
+      // setPrePendingCallPromise(device?.interruptionFromUser());
```

Biome (1.9.4)

[error] Change to an optional chain.

Unsafe fix: Change to an optional chain.


(lint/complexity/useOptionalChain)